migration/src/main/java/com/intershop/customization/migration/pfconfigurationfs/MigrateConfigResources.java

```diff
@@ -66,50 +92,48 @@ public void migrate(Path cartridgeDir)
 
                                  targetFile.toFile().getParentFile().mkdirs();
 
-                                 // resource files must be cinverted @see CfgResourceConverter
                                  String targetType = "";
-                                 if (targetFileName.endsWith("transport.resource"))
-                                 {
-                                     targetType = "transport";
-                                 }
-                                 else if (targetFileName.endsWith("application.resource"))
-                                 {
-                                     targetType = "application";
-                                 }
-                                 else if (targetFileName.endsWith("usr.resource"))
-                                 {
-                                     targetType = "user";
-                                 }
-                                 else if (targetFileName.endsWith("mngdsrvc.resource"))
-                                 {
-                                     targetType = "service";
-                                 }
-                                 else if (targetFileName.endsWith("dmnprfrnc.resource"))
+                                 // resource files must be cinverted @see CfgResourceConverter
+                                 if ((targetFileName.endsWith(".resource")) && (-1 < targetFileName.lastIndexOf('_')))
                                  {
-                                     targetType = "domain";
+                                     targetType = targetFileName.substring(targetFileName.lastIndexOf('_') + 1,
+                                                     targetFileName.lastIndexOf(".resource"));
                                  }
                                  if (!targetType.isEmpty())
                                  {
                                      // convert resource file to properties file
                                      String targetName = targetFile.toFile().getAbsolutePath();
                                      targetName = targetName.replace(".resource", ".properties");
                                      Path target = Paths.get(targetName);
+                                     
                                      convertResourceFile(targetType, source, target);
```

Comment:
hides a possible exception and therefor it is possible that the operation will be tracked as SUCCESS. Please double check.